src/ProductConstructionService/ProductConstructionService.Api/Program.cs

```diff
@@ -38,4 +42,12 @@
 
 app.MapControllers();
 
+// When running locally, create the workitem queue, if it doesn't already exist
+if (app.Environment.IsDevelopment())
+{
+    var queueServiceClient = app.Services.GetRequiredService<QueueServiceClient>();
+    var queueClient = queueServiceClient.GetQueueClient(app.Configuration[QueueConfiguration.PcsJobQueueName]);
```

Comment:
Can you use the const here too?

Reply:
It's there, or maybe I'm missing something?